# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../ebtables/ebtables-makefile-prefix.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ruN ebtables-v2.0.9-2-orig/Makefile ebtables-v2.0.9-2/Makefile
--- ebtables-v2.0.9-2-orig/Makefile	2010-02-03 22:17:45.000000000 +0100
+++ ebtables-v2.0.9-2/Makefile	2010-12-07 11:34:28.650696422 +0100
@@ -7,9 +7,10 @@
 PROGDATE:=June\ 2009
 
 # default paths
-LIBDIR:=/usr/lib
-MANDIR:=/usr/local/man
-BINDIR:=/usr/local/sbin
+PREFIX:=/usr/local
+LIBDIR:=$(PREFIX)/lib
+MANDIR:=$(PREFIX)/man
+BINDIR:=$(PREFIX)/sbin
 ETCDIR:=/etc
 INITDIR:=/etc/rc.d/init.d
 SYSCONFIGDIR:=/etc/sysconfig
